thunder/tests/opinfos.py

```diff
@@ -6152,6 +6152,32 @@ def sample_generator(op, device, dtype, requires_grad, **kwargs):
 nn_ops.append(max_pool3d_opinfo)
 
 
+def one_hot_sample_generator(op, device, dtype, requires_grad, **kwargs):
+    make = partial(make_tensor, device=device, dtype=torch.long, requires_grad=requires_grad)
+
+    test_shapes = [
+        (10,),
```

Comment:
Let's add a tensor with no dimensions (its shape is `()`) and a tensor with no elements (like `(0, 512)`, too

Reply:
Yeah, empty tensors + 0-dim tensors are very important. The underlying ops dispatch to PyTorch native implementations but these were written by me before the widespread adoption of 0-dim tensors... So we'd better double check that PyTorch as well does the right thing here...

Reply:
@nikitaved I've added an empty input `(, 512)` as you guys suggested (:

Reply:
@shaharelys , what about scalar inputs, the inputs with no dimension?

Reply:
Hey @nikitaved! Sorry for the delay. I did not add these. Should we also add these now?